### ISSUE TITLE
websocket_client_ssl example:  prints "Hello world!a"

### ISSUE THREAD

**Issue by Ivers87** [boostorg/beast#719]
I expect exactly the same string, as i sent to "echo.websocket.org", but it returns 'a'
in the end of my string. 

Am i doing something wrong? Thank you!

**Comment by Ivers87** [comment]
Oh, it should be 
ws.write(boost::asio::buffer(std::string("Hello, world!")), ec);

That way it works well.

**Comment by vinniefalco** [comment]
Yep, `boost::asio::buffer` will treat string literals as an array so it will include the null terminator, which is not what you want. Is there anything else I can help with or can we close the issue? Thanks!

**Comment by Ivers87** [comment]
Yeah, we can close the issue. Congraduation with entry into Boost.

**Comment by vinniefalco** [comment]
Thank you very much!!
